integration/containerd/cri/integration-tests.sh

```diff
@@ -327,9 +332,18 @@ main() {
 	# make sure cri-containerd test install the proper critest version its testing
 	rm -f "${CRITEST}"
 
-	pushd "${GOPATH}/src/${cri_containerd_repo}"
+	go get ${cri_repo}
+	pushd "${GOPATH}/src/${cri_repo}"
+
+	git reset HEAD
+	git checkout master
```

Comment:
here you shouldn't use the master branch, instead you should use the 1.3.10 corresponding cri branch.